include/ftl/fiber.h

```diff
@@ -192,85 +186,85 @@ class Fiber {
 };
 
 #if defined(FTL_FIBER_STACK_GUARD_PAGES)
-	#if defined(FTL_OS_LINUX) || defined(FTL_OS_MAC) || defined(FTL_iOS)
-		inline void MemoryGuard(void *memory, size_t bytes) {
```

Comment:
Similar argument here. This block should be indented to follow the logical flow. Otherwise, it's hard to see where the `#if` ends